packages/phone-number-privacy/combiner/src/common/metrics.ts

```diff
@@ -0,0 +1,47 @@
+import * as client from 'prom-client'
+const { Counter, Histogram } = client
+
+client.collectDefaultMetrics()
+
+export const Counters = {
+  requests: new Counter({
```

Comment:
Can we pivot this by API request? (getDistributedBlindedSalt, getBlindedMessageSig, etc)

Reply:
Since we'll be using log based metrics, the new structured logging changes that add the API endpoint and sessionID to all logs should allow us to do this